ILSpy.ReadyToRun/ReadyToRunLanguage.cs

```diff
@@ -69,7 +70,7 @@ public override ProjectId DecompileAssembly(LoadedAssembly assembly, ITextOutput
 		public override void DecompileMethod(IMethod method, ITextOutput output, DecompilationOptions options)
 		{
 			PEFile module = method.ParentModule.PEFile;
-			R2RReader reader = GetReader(module);
+			R2RReader reader = GetReader(null, module);
```

Comment:
I can't figure out how to get from a given `IMethod` object back to a `LoadedAssembly` object instance, is it possible?